### ISSUE TITLE
0.4.0 Getting Interpolation-only expressions are deprecated warning with terraform 0.13.5

### ISSUE THREAD

**Issue by apogrebnyak** [nozaq/terraform-aws-remote-state-s3-backend#31]
Here is the warning

```
Warning: Interpolation-only expressions are deprecated

  on .terraform/modules/remote_state/bucket.tf line 143, in data "aws_iam_policy_document" "state_force_ssl":
 143:       "${aws_s3_bucket.state.arn}",
```

Terraform required_version is set to `>=0.12`.  Because of this, the interpolation can be removed.

**Comment by nozaq** [comment]
fixed in #29

**Comment by nozaq** [comment]
published a fix as v0.4.1
